### ISSUE TITLE
Missing brick / block geometry (suggestion)

### ISSUE THREAD

**Issue by cloudrambler** [tsunamayo/Starship-EVO#2903]
Just started playing Starship-EVO a week ago and I'm hooked. Very well done Tsunamayo!

Been working on a ship concept, seeing what can be done with the stretchy blocks / bricks. Mostly I can create the necessary forms and surfaces, or figure out a variation that maintains the theme. However there is one logical base geometry type that is missing from the set and without it many continuous arrangements are simply not possible.

The attached image was found online. I don't know the origin so cannot credit it. I think it must be for another game using the same geometry set. In any case the illustration makes my case better than I can explain. Apologies if this request has been made before, but it would add value to the game if this final form could be included. Thank you.

![85524354-62b14600-b608-11ea-9b64-acbeb4cb7836](https://user-images.githubusercontent.com/70821082/92313616-bc010280-efc5-11ea-935f-a15e2c132d73.png)

**Comment by Hecateus** [comment]
the somewhat similar spaceship building game, Avorion, calls these Twisted Corners.

**Comment by tsunamayo** [comment]
Yes it have been requested many time, issue is making collision shape and symmetry. Cheers
